config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Locations::Engine.routes.draw do
+  resources :floors
```

Comment:
there was a location code called "new" and because we are the code is used as the id in the URL, `/holding_locations/new` presented a conflict between the existing location and the form to create a new one.